.github/workflows/test.yml

```diff
@@ -8,11 +8,12 @@ on:
 
 env:
   GO111MODULE: on
+  BUILDKIT_HOST: docker-container://buildkitd
 
 jobs:
   build:
     name: Build
-    runs-on: ubuntu-latest
+    runs-on: buildkitd
```

Comment:
Wait what is this magic?

Reply:
custom builder on digital ocean for better caching builds, here is the script we used to genreate the builder:
https://github.com/openllb/hlb/pull/133/commits/90b70c2b07e95d01c3699ddb8d71b12cbeea81ee